wallet/wallet.go

```diff
@@ -613,6 +613,89 @@ func (w *Wallet) LoadActiveDataFilters(n NetworkBackend) error {
 	return nil
 }
 
+// CommittedTickets takes a list of tickets and returns a filtered list of
+// tickets that are controlled by this wallet.
+func (w *Wallet) CommittedTickets(tickets []*chainhash.Hash) ([]*chainhash.Hash, error) {
+	hashes := make([]*chainhash.Hash, 0, len(tickets))
+	// Verify we own this ticket
+	err := walletdb.View(w.db, func(dbtx walletdb.ReadTx) error {
+		for _, v := range tickets {
+			// Make sure ticket exists
+			txmgrNs := dbtx.ReadBucket(wtxmgrNamespaceKey)
+			addrmgrNs := dbtx.ReadBucket(waddrmgrNamespaceKey)
```

Comment:
let's pull these two namespace variables outside of the loop

Reply:
Oops, yeah that was dumb. Fixed now.